File: m/rls/stlow.r
file	st_low - lowercase string
include	rid:rider
include	rid:stdef
include	rid:chdef

code	st_low - lowercase line

  func	st_low
	str : * char
	()  : * char
  is	ptr : * char~ = str
	spin while (*ptr++ = ch_low (*ptr)) ne
	reply str
  end


File: m/rls/rlb/stlow.c
/* file -  st_low - lowercase string */
#include "m:\rid\rider.h"
#include "m:\rid\stdef.h"
#include "m:\rid\chdef.h"
/* code -  st_low - lowercase line */
char *st_low(
char *str )
{ char *ptr = str;
  while ( (*ptr++ = ch_low (*ptr)) != 0) {;}
  return ( str);
} 
